internal/commands/listen.go

```
@@ -55,59 +55,67 @@ func runListen(c *cli.Context) error {
-
-	file, err := os.OpenFile(filename, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0666)
+	file, err := os.CreateTemp("", "ratiocheck-*.html")
 	if err != nil {
 		render.Status(r, http.StatusInternalServerError)
 		render.JSON(w, r, map[string]string{"error": err.Error()})
-		log.Printf("[%s] %e", r.Header.Get("X-Request-Id"), err)
+		log.Printf("[%s] %v", r.Header.Get("X-Request-Id"), err)
 		return
 	}
+	filename := file.Name()
+	defer os.Remove(filename)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)